packages/playground/big/src/App.svelte

```diff
@@ -13,7 +13,9 @@
 <nav>
 	<h2>This is a huge app</h2>
 	{#each Object.entries(pages) as [name, cmp] (cmp)}
-		<a class="App-link" href on:click|preventDefault={go(cmp)}>{name.replace('_', ' ')}</a>
+		<a class="App-link" href={`?page=${name}`} on:click|preventDefault={go(cmp)}
```

Comment:
I'm not sure but is `href` correct? After adding `jsconfig.json`, type errors occurred, so I’ve added a temporary href.

Reply:
this is an old test app with many generated components made by @rixo  i assume that because on:click calls `go` the href isn't wanted, if you need to use a value it could be`href="#"` or sth inert that doesn't mess with the basic "routing" in the app

Reply:
`#` shows a compiler warning.

![image](https://github.com/user-attachments/assets/6d9833d6-25f9-484a-abe4-b29e032b32e7)